### PR TITLE
chore(base): Make useSystemUnzip option configurable via process environment

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

https://github.com/appium/appium-base-driver/pull/537 to 2.0

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by mykola-mokhnach — APPROVED** [review]

File: packages/base-driver/lib/basedriver/helpers.js [hunk 1]
```diff
@@ -322,15 +322,16 @@ async function unzipApp (zipPath, dstRoot, supportedAppExtensions) {
...
+    const useSystemUnzip = _.isEmpty(useSystemUnzipEnv)
+      || !['0', 'false'].includes(_.toLower(useSystemUnzipEnv));
     /**
      * Attempt to use use the system `unzip` (e.g., `/usr/bin/unzip`) due
      * to the significant performance improvement it provides over the native
      * JS "unzip" implementation.
      * @type {import('@appium/support/lib/zip').ExtractAllOptions}
      */
-    const extractionOpts = {
-      useSystemUnzip: !system.isWindows(),
```
Comment (mykola-mokhnach) [inline packages/base-driver/lib/basedriver/helpers.js:332]: was `@appium/support` module version bumped to use this functionality?

**Comment by boneskull** [comment]
Is there precedent for adding env vars to control this sort of thing?  Would it be better as a flag?

**Comment by boneskull** [comment]
@KazuCocoa When creating PRs, please create a branch `kazucocoa/<branchname>` in _this_ repo.  This is why the FOSSA scan is failing (no access to API key).

I will modify the build to avoid running FOSSA checks for PRs with their heads in forks... if you can do that.

**Comment by KazuCocoa** [comment]
oh, i see > branch name

File: packages/base-driver/lib/basedriver/helpers.js (hunk 1 above)
Comment (KazuCocoa) [reply packages/base-driver/lib/basedriver/helpers.js:332]: yea. the package.json already has the latest `@appium/support` package.

**Comment by KazuCocoa** [comment]
> Is there precedent for adding env vars to control this sort of thing? Would it be better as a flag?

Afaik, some of. For example, appium-chromedriver allows users to set download URL via an environment variable.
But yes, probably server arg or capabilities is also good

**Pull request merged**